components/alert-button/alert.css

```diff
@@ -0,0 +1,68 @@
+alert-container{
+    display:flex;
+    flex-flow:wrap column;
+    align-items:flex-start;
+    margin-left:0;
+    border:1px solid grey;
+    padding:25px;
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
+      
+}
+
+.alert{
+    width:400px;
+    padding: 10px;
+    text-align:left;
+    flex-basis:50px;
+    margin-bottom:0.5rem;
```

Comment:
Just a suggestion use px for units less than 1rem

Reply:
ok thanks for suggestion